docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -780,7 +781,34 @@ directly or used in conditional statements.
 - `$file_size`
   : Size of the file for this row.
 
+### Replace tables
```

Comment:
How about moving to before `ALTER TABLE EXECUTE`?

Reply:
Sure .. that works too.